tailwind.config.js

```diff
@@ -18,9 +18,23 @@ export default {
       }
     },
   },
-  plugins: [debugPlugin()],
+  plugins: [debugPlugin(), autocompleteCustomComponentsPlugin()], 
 };
 
+/**
+ * Custom plugin that provide auto-complete support for the listed classes.
+ *
+ * The style definitions can be found in `index.css` under the `@layer components {` section
+ * 
+ */
+function autocompleteCustomComponentsPlugin() {
```

Comment:
With this you can now access the custom tailwind components in intellisense

<img width="541" alt="image" src="https://github.com/refstudio/refstudio/assets/174127/db772912-37d1-4274-b7d8-ceb5cdd80f0e">